### PR TITLE
feat: Implements param lockSettingsHideViewersCursor (Backend only)

### PR DESCRIPTION
**Pull request by gustavotrott**
Backend portion of #14636

Implements `lockSettingsHideViewersCursor` param in API /create.

And in `bigbluebutton.properties`:
image

Let `hideViewersCursor` available for html5 in `lockSettingsProps`:
image

### CODE REVIEW

File: bigbluebutton-web/grails-app/conf/spring/resources.xml [hunk 1]
```diff
@@ -171,7 +171,7 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
         <property name="muteOnStart" value="${muteOnStart}"/>
         <property name="allowModsToUnmuteUsers" value="${allowModsToUnmuteUsers}"/>
         <property name="allowModsToEjectCameras" value="${allowModsToEjectCameras}"/>
-        <property name="breakoutRoomsEnabled" value="${breakoutRoomsEnabled}"/>
+        <property name="breakoutRoomsEnabled" value="${breakoutRoomsEnabled:true}"/>
```
Comment (antobinary) [inline bigbluebutton-web/grails-app/conf/spring/resources.xml:174]: Why did you need to pass `true` for `breakoutRoomsEnabled`? Is that instead of an entry in bigbluebutton.properties?

Comment (gustavotrott) [reply bigbluebutton-web/grails-app/conf/spring/resources.xml:174]: It is there to avoid warnings..

Since #14599, `breakoutRoomsEnabled` was removed from config by default.
So this set `true` as alternative value when `breakoutRoomsEnabled` is absent from config file.
